21-Hello-World/01-Prelude-ish/02-Prelude/08-Control-Flow--Functor-to-Monad/03-Applicative.md

```diff
@@ -1,15 +1,15 @@
-# Applyicative
+# Applicative
```

Comment:
Could you remove this from your PR? It's technically a breaking change and Miles has already submitted a PR for it in #463, which I don't plan to merge until the next PS release is made.

Otherwise, everything else looks good. Thanks for the PR!